common/wuanalysis/anacommon.hpp

```diff
@@ -19,6 +19,7 @@
 #define ANACOMMON_HPP
 
 #include "jliball.hpp"
+#include "workunit.hpp"
```

Comment:
Why is this needed?

Reply:
It doesn't need to be here - I'm removing.